tests/workflows/basic_await_attributes/tests/test_workflow.py

```diff
@@ -14,4 +14,4 @@ def test_workflow__happy_path():
 
     # THEN the Workflow completes successfully
     assert final_event.name == "workflow.execution.fulfilled"
-    assert final_event.outputs.final_value == 2
+    assert final_event.outputs.final_value == 3
```

Comment:
This test was also busted. An AWAIT_ATTRIBUTES node should be invoked three times if there are three dependencies and the attributes are set for all three